spec/unit/analytics/ga4/formatter_spec.rb

```diff
@@ -0,0 +1,116 @@
+require "spec_helper"
+require_relative "../../../../lib/analytics/ga4/formatter"
+require_relative "../../../../lib/analytics/ga4/page_data"
```

Comment:
```suggestion
require "analytics/ga4/formatter"
require "analytics/ga4/page_data"
```

same for other tests.